requirements-minimal.txt

```diff
@@ -6,7 +6,6 @@ scipy>=1.4.1
 h5py>=2.10.0
 hdmf>=3.4.7
 pynwb>=1.4.0;python_version>='3.8'
```

Comment:
When was the `OnePhotonSeries` added to PyNWB? We should pin to that as the minimal version now

Reply:
OK, sounds good